astroquery/gaiatap/core.py

```diff
@@ -0,0 +1,383 @@
+        """Launches a synchronous job
+        TAP & TAP+
+        
+        Parameters
+        ----------
+        query : str, mandatory
+            query to be executed
+        output_file : str, optional, default None
+            file name where the results are saved if dumpToFile is True. 
+            If this parameter is not provided, the jobid is used instead
+        output_format : str, optional, default 'votable'
+            results format
+        verbose : bool, optional, default 'False' 
+            flag to display information about the process
+        dump_to_file : bool, optional, default 'False'
+            if True, the results are saved in a file instead of using memory
+        upload_resource: str, optional, default None
+            resource to be uploaded to UPLOAD_SCHEMA
+        upload_table_name: str, required if uploadResource is provided, default None
+            resource temporary table name associated to the uploaded resource
+
+        Returns
+        -------
+        A Job object
+        """
+        return self.__gaiatap.launch_sync_job(query, name=name, output_file=output_file, output_format=output_format, verbose=verbose, dump_to_file=dump_to_file, upload_resource=upload_resource, upload_table_name=upload_table_name)
+    
+    def launch_async_job(self, query, name=None, output_file=None, output_format="votable", verbose=False, dump_to_file=False, background=False, upload_resource=None, upload_table_name=None):
+        """Launches an asynchronous job
+        TAP & TAP+
+        
+        Parameters
+        ----------
+        query : str, mandatory
+            query to be executed
+        output_file : str, optional, default None
+            file name where the results are saved if dumpToFile is True. 
+            If this parameter is not provided, the jobid is used instead
+        output_format : str, optional, default 'votable'
+            results format
+        verbose : bool, optional, default 'False' 
+            flag to display information about the process
+        dump_to_file : bool, optional, default 'False'
+            if True, the results are saved in a file instead of using memory
+        background : bool, optional, default 'False'
+            when the job is executed in asynchronous mode, this flag specifies whether 
+            the execution will wait until results are available
+        upload_resource: str, optional, default None
+            resource to be uploaded to UPLOAD_SCHEMA
+        upload_table_name: str, required if uploadResource is provided, default None
+            resource temporary table name associated to the uploaded resource
+
+        Returns
+        -------
+        A Job object
+        """
+        return self.__gaiatap.launch_async_job(query, name=name, output_file=output_file, output_format=output_format, verbose=verbose, dump_to_file=dump_to_file, background=background, upload_resource=upload_resource, upload_table_name=upload_table_name)
```

Comment:
Minor style issue: could you make sure these long lines are wrapped at 80 characters, or at least close to 80?